src/world_interface/data.h

```diff
@@ -56,41 +56,24 @@ enum class motorid_t {
 	shoulder,
 	elbow,
 	forearm,
-	differentialRight,
-	differentialLeft,
+	wrist,
 	hand
 };
 
 /** @brief the mounted peripheral on the robot. */
-enum class mountedperipheral_t {
-	none,
-	arm,
-	scienceStation,
-	lidar
-};
+enum class mountedperipheral_t { none, arm, scienceStation, lidar };
 
-enum class jointid_t {
-	armBase,
-	shoulder,
-	elbow,
-	forearm,
-	differentialRoll,
-	differentialPitch,
-	hand,
-	drill_arm
-};
+enum class jointid_t { armBase, shoulder, elbow, forearm, wrist, hand, drill_arm };
```

Comment:
**non-blocking nitpick:** Do we want jointid_t and mountedperipheral_t to be single line enums or is it clang-format's doing? If single line enums are fine, it would be nice to add a newline between jointid_t and all_jointid_t for readability. If it's clang-format doing it, there is an option we can set in clang-format to enforce multi line enums.

Reply:
Good point, multiline enums are more readable imo. Fixed it with new clang-format rule.